File: assets/rt/RealSystem.java
package rt;

public final class RealSystem implements base.caps._System_0 {
  @Override public RealSystem iso$mut() { return this; }
  @Override public RealSystem self$mut() { return this; }
  @Override public rt.IO io$mut() {
    return rt.IO.$self;
  }
  @Override public rt.CapTry try$mut() {
    return new rt.CapTry();
  }
  @Override public base.caps.RandomSeed_0 rng$mut() {
    return rt.Random.SeedGenerator.$self;
  }
}

File: assets/rt-js/RealSystem.js
import { rt$$IO } from './IO.js';

export class RealSystem {
  iso$mut$0() { return this; }
  self$mut$0() { return this; }

  io$mut$0() {
    return rt$$IO.$self;
  }

  // try$mut$0() {
  //   return new CapTry(); // implement like in your Java runtime
  // }
  //
  // rng$mut$0() {
  //   return Random.SeedGenerator.$self; // stub
  // }
}